docs/system-admin-guide/integrations/nextcloud/README.md

```diff
@@ -1,100 +1,183 @@
+
+![Integration successfully set up on the Nextcloud end, three green checks visible](3_2_05-NC_Success.png)
 
-![OpenProject_success](3_4_00-OAuth-OP_Success.png)
+#### 5. Add your new Nextcloud file storage to a project
 
-![Nextcloud_success](3_2_05-NC_Success.png)
+The integration is now complete, and your OpenProject and Nextcloud instances are ready to share information. The next step is to make the Nextcloud file storage you just created available to individual projects.
 
-**5. Enable the file storages module and activate it in the desired project** - in OpenProject
+To do so, navigate to any existing project in your OpenProject instance and click on  **Project settings** **→ Modules**. There, active the **File Storages** module by clicking on the checkbox next to it and then clicking on **Save**.
 
-Now that the integration setup is complete and ready to use, there is just one more step for you as administrator: in every project in which you want to use the integration you need to activate the *File storages* module and add your recently configured file storage to the project.
+![List of modules activated for a project](project-modules.png)
 
-To activate the module, go to the **Project settings** in the desired project and access the **Modules** entry in the side menu. There, activate the *File storages* module. Once this is done, a new menu entry called **File storages** appears at the bottom of the side menu, where you can select your new Nextcloud integration.
+Click on this new **Files storages** menu entry. It will take you to a page titled **File storages available in this project**, which is normally empty.
 
-![module_activation](Settings_modules.png)
+Click on **+ Storage**.
 
-![settings_files_storages](Settings_files_storages.png)
+![List of file storages available to a project, empty](file-storages-available-in-project.png)
 
-## How to reset your OAuth values
+In the page that follows, under, make sure your newly-created Nextcloud file storage is selected under **Storage**. If you only have one Nextcloud file storage set up (which is the most likely scenario), it should already be selected by default.
 
-If you need to reset the values of the Nextcloud integration (as an administrator), you can always reset the OAuth values from both sides of the integration by clicking on the **Reset OAuth values** (in Nextcloud) or **Replace OAuth values** (in OpenProject) buttons.
+![Add your new file storage to a project](add-file-storage-to-project.png)
+
+Click on **Add** to add your new Nextcloud file storage to this project.
+
+The Nextcloud file storage is now available to all work packages in this project. 
+
+> **Note:** For information on how to use the file storage (link Nextcloud user accounts at a user level, link files to a work package, view and download linked files, unlink files), please read our [Nextcloud integration user guide](../..//user-guide/nextcloud-integration/).
```

Comment:
Invalid path to "Nextcloud integration user guide" 
Please change to `../../../user-guide/nextcloud-integration/`

Reply:
Done.